runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -290,6 +291,13 @@ else if (result.equals("were defeated"))
 
 			setKc("Barrows Chests", kc);
 		}
+		
+		matcher = GAUNTLET_PATTERN.matcher(message);
+		// TODO: set lastBossKill in kc command instead
```

Comment:
what is this referring to?

Reply:
I had a hunch that the messages were being sent as one, and that I could do that. Meant the kc message, but either way it's not relevant anymore. Must have left that in on accident, apologies.